### DISCUSSION TITLE
Section_pass/fail sound is missed

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#18829] [Q&A] [answered]
More than 2 years ago I created a request to add the classic "section_pass/fail" element - https://github.com/ppy/osu/issues/9783. I also came across this - https://github.com/ppy/osu/issues/14082.  It seems to me that if this function itself has changed a lot, then the classic sound (or updated) is not enough for it. It is not considered missing or is it needed/planned to be added?

**Comment by frenzibyte** [comment, marked as answer]
I don't believe this has been missed or anything? The issue thread is still there and open, and will eventually be added.
